src/components/Gallery.jsx

```diff
@@ -21,17 +21,13 @@ class Gallery extends React.Component {
     this.setState({ 'showModal': !this.state.showModal })
   }
 
-  modalHover() {
-    return;
-  }
-
   render () {
     return (
       <div>
         {this.props.style.photos.map((photo) => (
```

Comment:
When mapping components it would be good to have a unique 'key' assignment for each component. This won't break anything but it prevents a react warning to appear in the console.